engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -176,15 +180,21 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
     }
 
     private void initRenderingSupport() {
-        frameBuffersManager = new FrameBuffersManager();
+        dynamicFBM = new DynamicFBM();
+        staticFBM = new StaticFBM();
+        shadowMapResolutionDependentFBM = new ShadowMapResolutionDependentFBM();
 
-        context.put(FrameBuffersManager.class, frameBuffersManager);
+        context.put(DynamicFBM.class, dynamicFBM);
+        context.put(StaticFBM.class, staticFBM);
+        context.put(ShadowMapResolutionDependentFBM.class, shadowMapResolutionDependentFBM);
```

Comment:
I'm not convinced these should be so public as to be available through the context. In my opinion only objects working within the renderer should obtain them, i.e. via constructor or injection.